### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by Biswa96**

### CODE REVIEW

**Comment by Biswa96** [comment]
GitHub Actions can not download source from invisible-mirror.net of diffstat. Need manual build.

File: mksh/PKGBUILD [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Maintainer: Masanori Shirayama <mashir43@gmail.com>
 
 pkgname=mksh
-_ver=59
+_ver=59c
```
Comment (lazka) [inline mksh/PKGBUILD:4]: needs "59.c" since "59" > "59c"

Comment (Biswa96) [reply mksh/PKGBUILD:4]: In next line, `_ver` is converted to correct `pkgver`.

**Comment by Biswa96** [comment]
Ping!

File: mksh/PKGBUILD (hunk 1 above)
Comment (lazka) [reply mksh/PKGBUILD:4]: oops

**Comment by lazka** [comment]
Pong!

**Pull request merged**